airbyte-webapp-e2e-tests/cypress/pages/helpers/table.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * Click on specific cell found by column name in desired table
+ * @param tableSelector - table selector
+ * @param columnName - column name
+ * @param connectName - cell text
+ */
+export const clickOnCellInTable = (tableSelector: string, columnName: string, connectName: string) => {
+  cy.contains(`${tableSelector} th`, columnName)
+    .invoke("index")
+    .then((value) => {
+      cy.log(`${value}`);
+      return cy.wrap(value);
+    })
+    .then((columnIndex) => {
+      cy.contains("tbody tr", connectName).find("td").eq(columnIndex).click();
+    });
+};
```

Comment:
Oh, yes, good catch! I missed that we have `common.ts`.  
Since they follow the same goal, I will move `clickOnCellInTable` to be more consistent. Thanks!